editor/src/components/canvas/controls/insert-mode-control-container.tsx

```diff
@@ -437,7 +438,16 @@ export class InsertModeControlContainer extends React.Component<
       let { element } = this.props.mode.subject
       if (this.isTextInsertion(element, insertionSubject.importsToAdd)) {
         element = this.getTextElementAutoSizeWithPosition(element, snappedMousePoint)
+      } else if (this.parentIsFlex(this.props.highlightedViews[0])) {
+        element = {
+          ...element,
+          props:
+            eitherToMaybe(
+              unsetJSXValueAtPath(element.props, createLayoutPropertyPath('position')),
```

Comment:
if I understand correctly, this here means that inserting an element to flex will not give it a position prop. 

if we want to keep a transitional behavior (allowing for any child to be pinned), we want to insert a `position: relative` here